src/test/java/org/skriptlang/skript/test/tests/syntaxes/events/PlayerElytraBoostEventTest.java

```diff
@@ -0,0 +1,74 @@
+package org.skriptlang.skript.test.tests.syntaxes.events;
+
+import ch.njol.skript.test.runner.SkriptJUnitTest;
+import com.destroystokyo.paper.event.player.PlayerElytraBoostEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Firework;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.easymock.EasyMock;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Constructor;
+
+public class PlayerElytraBoostEventTest extends SkriptJUnitTest {
+
+	private Player player;
+	private Firework firework;
+
+	@Before
+	public void setUp() {
+		player = EasyMock.niceMock(Player.class);
+		EntityType entityType = null;
+		if (EntityType.valueOf("FIREWORK") != null) {
+			entityType = EntityType.valueOf("FIREWORK");
+		} else if (EntityType.valueOf("FIREWORK_ROCKET") != null) {
+			entityType = EntityType.valueOf("FIREWORK_ROCKET");
+		}
```

Comment:
```suggestion
		EntityType entityType = EntityType.valueOf("FIREWORK");
		if (entityType == null) {
			entityType = EntityType.valueOf("FIREWORK_ROCKET");
		}
```
To simplify a little